providers/netcore/src/lib/netcore.provider.ts

```diff
@@ -71,4 +84,61 @@ export class NetCoreProvider implements IEmailProvider {
       code: CheckIntegrationResponseEnum.SUCCESS,
     };
   }
+
+  getMessageId(body: any | any[]): string[] {
```

Comment:
I guess this is the same as message_id we get in the response from netcore when we send the email?

Reply:
For NetCore, it's TRANSID.

Reply:
okay then we need to change message_id in the send method?

Reply:
Sure we can send that!

Reply:
did you check so TRANSID exists on the response on send?

Reply:
Yes